src/common/libkvs/kvs_commit.c

```diff
@@ -163,6 +163,15 @@ static int decode_response (flux_future_t *f, const char **rootrefp,
     return 0;
 }
 
+int flux_kvs_commit_get_rootref (flux_future_t *f, const char **rootref)
+{
+    if (!f || !rootref) {
+        errno = EINVAL;
+        return -1;
+    }
+    return decode_response (f, rootref, NULL);
+}
+
```

Comment:
Correct me if I'm wrong but this appears to not be used anymore by the sync implementation in this PR.  It's still handy though I guess - split to another PR?

Reply:
yeah, was thinking of splitting it off anyways, so i'll do that right now!